client/src/app/core/services/keycloak/keycloak.service.ts

```diff
@@ -32,12 +32,15 @@ export class KeycloakService {
     }
   }
 
+  loggedIn() {
```

Comment:
nitpick: please use isLoggedIn to make clear this function returns a boolish value